common/utils/src/main/resources/error/error-conditions.json

```
@@ -2072,6 +2072,12 @@
     ],
     "sqlState" : "22003"
   },
+  "INVALID_ATTRIBUTE_NAME_SYNTAX" : {
+    "message" : [
+      "Syntax error in attribute name: <name>."
```

Can you provide users some hints about what could be wrong like and what to check:
```suggestion
      "Syntax error in the attribute name: <name>. Check that backticks appear in pairs, a quoted string is a complete name part and use a backtick only inside quoted name parts."
```

---

Looks way better. Thanks for the suggestion!